project/Build.scala

```diff
@@ -370,9 +370,12 @@ object Build {
             Seq(s"-Xplugin:$jar")
           }
         }
-      )
+      ).withScalaJUnitMixinPlugin
     }
 
+    def withScalaJUnitMixinPlugin: Project =
+      project.settings(ScalaJUnitMixinPlugin.projectSettings)
```

Comment:
I suggest you inline that definition right inside `withScalaJSJUnitPlugin`, which is the only user of this method.
Moreover, I suggest you inline `ScalaJUnitMixinPlugin.projectSettings right there within `withScalaJSJUnitPlugin` as well, and get rid of `ScalaJUnitMixinPlugin`.